core/src/main/scala/codesearch/core/Main.scala

```diff
@@ -16,7 +16,7 @@ object Main extends IOApp {
   final case class Params(
       updatePackages: Boolean = false,
       downloadMeta: Boolean = false,
-      initDB: Boolean = false,
+      initDB: Boolean = true,
```

Comment:
I think it's not enough to consider this task completed. At least we should delete the `make table` task from code, Makefile, Readme, docker files, etc.

Reply:
Ok, i removed usages of `-i` flag

Reply:
Please make sure that you removed all usages of this command. The most important place where it mentioned is [docker-stack-compose.yml](https://github.com/aelve/codesearch/blob/255-fix-default-db-init/docker/docker-stack-compose.yml#L49)